### PR TITLE
Develop apd sprint1 textlist bugfix

### PR DESCRIPTION
**Pull request by jleong01**
Updated dialog to fix display

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/coursepage/body.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!--
+  ~ <%--
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/coursepage/body.html]: no need of this file. please remove. do the changes into the basepage/body.html in the separate PR

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/coursepage/content.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<div data-sly-resource="${'par' @resourceType='wcm/foundation/components/responsivegrid'}"></div>
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/coursepage/content.html]: no need of this file. please remove.

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/coursepage/body.html (hunk 1 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/coursepage/body.html]: file removed

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/coursepage/content.html (hunk 2 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/coursepage/content.html]: file removed

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/banner/_cq_dialog/.content.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <title
+                                jcr:primaryType="nt:unstructured"
+                                jcr:title="Banner"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/fieldset">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <mobileimage
+                                        jcr:primaryType="cq:Widget"
+                                        sling:resourceType="granite/ui/components/foundation/form/pathbrowser"
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/banner/_cq_dialog/.content.xml]: please use coral pathfield

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/banner/_cq_dialog/.content.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                                        fieldLabel="Mobile image path"
+                                        name="./mobileimage"/>
+                                    <desktopimage
+                                        jcr:primaryType="cq:Widget"
+                                        sling:resourceType="granite/ui/components/foundation/form/pathbrowser"
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/banner/_cq_dialog/.content.xml]: please use coral pathfield

Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/banner/_cq_dialog/.content.xml]: updated

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/banner/_cq_dialog/.content.xml (hunk 3 above)
Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/banner/_cq_dialog/.content.xml]: updated to coral pathfield

**Pull request merged**